go/chat/search/indexer.go

```diff
@@ -454,6 +454,10 @@ func (idx *Indexer) allConvs(ctx context.Context, uid gregor1.UID) (map[string]t
 			chat1.ConversationStatus_FAVORITE,
 			chat1.ConversationStatus_MUTED,
 		},
+		MemberStatus: []chat1.ConversationMemberStatus{
```

Comment:
Why bother with this?

Reply:
`RESET` is included in the default inbox query, might speed things up